src/foam/u2/FragmentedTextField.js

```diff
@@ -95,19 +97,36 @@ foam.CLASS({
       this.numOfParts = this.delegates.length;
 
       return this;
+    },
+    function processData(data, index, arr) {
```

Comment:
So this looks like it also works if I'm in the second box and decide to "paste" the rest of the number, is this correct?

Reply:
yeah it made sense to me if the data was half filled, and you paste from some empty box it would work similarly and fill the data starting from that box and onwards